src/libstore/http-binary-cache-store.cc

```diff
@@ -23,6 +23,8 @@ class HttpBinaryCacheStore : public BinaryCacheStore
 
 public:
 
+    const Setting<bool> cache{this, true, "cache", "Whether to cache narinfo metadata for this binary cache."};
```

Comment:
should probably move to `src/libstore/binary-cache-store.hh` ?

Also I guess this is useful per binary cache but it's possible to set something globally through `--option narinfo-cache-{positive/negative}-ttl 0` that disables cache as well.

Reply:
narinfo caching doesn't make sense for file binary caches, but I could move it